### PR TITLE
Mm/testing

### PR DESCRIPTION
**Pull request by MarchandMD**
# What does this PR do? 
- adds `supertest` to package.json
- begins building a test suite
- updates readme

# How does this PR accomplish that? 
- see git diff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,6 @@
 *** https://www.markdownguide.org/basic-syntax/#reference-style-links
 -->
 
-[![Contributors][contributors-shield]][contributors-url]
-[![Forks][forks-shield]][forks-url]
-[![Stargazers][stars-shield]][stars-url]
-[![Issues][issues-shield]][issues-url]
-[![MIT License][license-shield]][license-url]
-
 <!-- PROJECT LOGO -->
```
Comment (MarchandMD) [inline README.md:22]: I didn't really like these badges at the top; I added a "contributors" section further down

File: README.md [hunk 2]
```diff
@@ -46,9 +46,9 @@
     ·
     <a href="https://github.com/dnvr-zero/dnvr-zero-fe">View Frontend source code</a>
     .
-    <a href="https://github.com/marchandmd/dnvr-zero/dnvr-zero-be/issues">Report Bug</a>
+    <a href="https://github.com/dnvr-zero/dnvr-zero-be/issues">Report Bug</a>
     ·
-    <a href="https://github.com/marchandmd/dnvr-zero/dnvr-zero-be/issues">Request Feature</a>
+    <a href="https://github.com/dnvr-zero/dnvr-zero-be/issues">Request Feature</a>
   </p>
```
Comment (MarchandMD) [inline README.md:52]: found dead links; fixed them

File: README.md [hunk 3]
```diff
@@ -86,6 +86,15 @@
 
 We're alumni of the Turing School of Software and Design, and this project is part of the New Grad Services program, which provides opportunity for recent grads to continue to practice existing skills, gain new tech stack, and build community around an open source project.
 
+<details>
+<summary>Click for more</summary>
+<p>We're (re)creating (or re-vitalizing) a site that encourages non-consumer leisure activities, creativity and exploring the habitat in which you find yourself. Create a "player", find a task to complete, and then sign up for that task. Submit proof once your done and gain sweet, sweet fake internet points to level up.</p>
```
Comment (MarchandMD) [inline README.md:91]: added a collapsible section with more boilerplate language about the project

File: README.md [hunk 4]
```diff
@@ -95,6 +104,8 @@ We're alumni of the Turing School of Software and Design, and this project is pa
 -   [![Express][Express]][express-url]
 -   [![React][react]][react-url]
 -   [![Node][Node]][node-url]
+-   [![TypeScript][TypeScript]][typescript-url]
+-   [![Bootstrap][Bootstrap]][bootstrap-url]
```
Comment (MarchandMD) [inline README.md:108]: added additional badges for the tech stack

File: README.md [hunk 5]
```diff
@@ -192,13 +203,53 @@ Distributed under the MIT License. See `LICENSE.txt` for more information.
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTACT -->
-
-## Contact
-
-Michael Marchand - MichaelDavidMarchand@gmail.com
-
-Project Link: [https://github.com/dnvr-zero](https://github.com/dnvr-zero)
+<!-- Contributors -->
```
Comment (MarchandMD) [inline README.md:206]: This is a markdown/html table showing our photos, names, and links to github and linkedin

**Review by msakr21 — APPROVED** [review]
Looking good.

**Pull request merged**